pywps/app/Process.py

```diff
@@ -278,10 +278,8 @@ def clean(self):
             if self._grass_mapset and os.path.isdir(self._grass_mapset):
                 LOGGER.info("Removing temporary GRASS GIS mapset: %s" % self._grass_mapset)
                 shutil.rmtree(self._grass_mapset)
-        except WindowsError as err:
-                LOGGER.error('Windows Error: %s', err)
         except Exception as err:
-                LOGGER.error('Unable to remove directory: %s', err)
+            LOGGER.error('Unable to remove directory: %s', err)
```

Comment:
`WindowsError` should be subclass of `Exception`, therefore the logic should remain the same. WindowsError is available on windows only - that caused some errors on other platforms (see #218)

this is actually core of the patch. other lines are just minor enhancements